.rubocop.yml

```diff
@@ -60,3 +60,10 @@ Naming/MethodParameterName:
 
 Naming/VariableNumber:
   CheckSymbols: false
+  Exclude:
+    - 'test/**'
+
+Metrics/BlockLength:
+  Exclude:
+    - 'test/**'
```

Comment:
test methods have large numbers of asserts which do not make sense to break into new blocks